regression/ansi-c/simplify-overflow/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE gcc-only
```

Comment:
The `#ifdef _MSC_VER` suggests that the test might work with goto-cl?

Reply:
Ah, thank you, that was a left-over from when I thought about the `__builtin_constant_p` approach. Fixed.